src/main/java/com/hillel/homework/hw1/SimpleReservationSystem.java

```diff
@@ -0,0 +1,99 @@
+package com.hillel.homework.hw1;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+public class SimpleReservationSystem {
+    public static void main(String[] args) throws IOException {
+
+        ArrayList<Table> tables = new ArrayList<>();
+        for (int i = 1; i <= 10; i++) {
+            tables.add(new Table(i));
+        }
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
bufferedReader can be static field, in this case you don't need to pass it to each method, it will be available from each method by itself